flexget/plugins/sites/newpct.py

```diff
@@ -64,7 +74,49 @@ def parse_download_page(self, url):
             torrent_id = torrent_id_prog.search(torrent_ids[0]).group(1)
             return 'http://www.newpct.com/torrents/{:0>6}.torrent'.format(torrent_id)
 
+    def search(self, task, entry, config=None):
+        if not config:
+            log.debug('NewPCT disabled')
+            return set()
+        log.debug('Search NewPCT')
+        url_search = 'http://www.newpct1.com/buscar'
+        results = set()
+        for search_string in entry.get('search_strings', [entry['title']]):
+            query = normalize_unicode(search_string)
+            query = re.sub(' \(\d\d\d\d\)$', '', query)
+            log.debug('Searching NewPCT %s' % query)
```

Comment:
Let the logger do the string interpolation:
`log.debug('Searching NewPCT %s', query)`